src/motor_hardware.cc

```diff
@@ -68,6 +69,8 @@ int32_t  g_odomLeft  = 0;
 int32_t  g_odomRight = 0;
 int32_t  g_odomEvent = 0;
 
+// 2x6S(12 V) Batteries connected in series
+const uint8_t SLA_AGM_CELLS = 6 * 2;
```

Comment:
I suppose you could use this but any of our batteries alone at 12V is 'low'.   I do see later you use 2.175 sort of number that would be 13V.  A nominal 12V gel cell is near 12.6V actually.
I leave this up to you to think about and you can leave this if you like except the 2.175 factor should be a factor to come out to more like 12.6 but again batteries brand new fresh totally charted can read 13v so up to you.